src/MeshIntegrals.jl

```diff
@@ -24,6 +24,7 @@ include("integral_aliases.jl")
 export lineintegral, surfaceintegral, volumeintegral
 
 # Integration methods specialized for particular geometries
+
```

Comment:
This is just a test to trigger CI and see if everything looks fine with CI and code coverage after the move.
```suggestion
```